elastic/beats
# [Heartbeat] Add heartbeat global trace path option

## What does this PR do?

Add `heartbeat.trace` path to dump events generated. This gives us the option to ppublish events to multiple sources.

## Checklist




## How to test this PR locally

Build heartbeat locally and specify `heartbeat.trace` path on `heartbeat.yml` and run monitors. Events should be appended to the file as they are generated.

## Related issues


-
